src/sentry/killswitches.py

```diff
@@ -37,9 +38,16 @@ def _update_project_configs(
     # or removed
     changed_project_ids = old_project_ids ^ new_project_ids
 
-    with click.progressbar(changed_project_ids) as ids:
-        for project_id in ids:
-            if project_id is not None:
+    if None in changed_project_ids:
+        with click.progressbar(length=Organization.objects.count()) as bar:
+            for org in Organization.objects.all():
+                schedule_invalidate_project_config(
+                    trigger="invalidate-all", organization_id=org.id, countdown=0
+                )
+                bar.update(1)
```

Comment:
I basically inlined the (now unused) `invalidate_all_project_configs` task here instead of calling it, it gives a progressbar and you know when they have all been scheduled instead of having to guess based on what you can observe in rabbitmq.  I'm a bit torn on this, I think I slightly prefer this and would choose to delete the task for now.